content/en/docs/concepts/configuration/manage-resources-containers.md

```diff
@@ -34,16 +34,20 @@ For example, if you set a `memory` request of 256 MiB for a container, and that
 a Pod scheduled to a Node with 8GiB of memory and no other Pods, then the container can try to use
 more RAM.
 
-If you set a `memory` limit of 4GiB for that container, the kubelet (and
-{{< glossary_tooltip text="container runtime" term_id="container-runtime" >}}) enforce the limit.
-The runtime prevents the container from using more than the configured resource limit. For example:
-when a process in the container tries to consume more than the allowed amount of memory,
-the system kernel terminates the process that attempted the allocation, with an out of memory
-(OOM) error.
+Both `CPU` and `memory` limits are applied by the kubelet (and
+{{< glossary_tooltip text="container runtime" term_id="container-runtime" >}}),
+and are ultimately enforced by the kernel. On Linux nodes, limits are enforced
+by {{< glossary_tooltip text="cgroups" term_id="cgroup" >}}). However, `CPU` and `memory` limits
+are enforced differently.
 
```

Comment:
I'll suggest you leave this as it is.
```suggestion
(OOM) error.
```

Reply:
this line was dropped because the paragraph was reworked, so I think we should keep it dropped

Reply:
Alright